VDF.Core/ScanEngine.cs

```diff
@@ -296,9 +314,9 @@ public sealed class ScanEngine {
 							try {
 								Image bitmapImage = Image.FromFile(entry.Path);
 								float resizeFactor = 1f;
-								if (bitmapImage.Width > 100 || bitmapImage.Height > 100) {
-									float widthFactor = bitmapImage.Width / 100f;
-									float heightFactor = bitmapImage.Height / 100f;
+								if (bitmapImage.Width > thumbnailWidth || bitmapImage.Height > thumbnailWidth) {
+									float widthFactor = bitmapImage.Width / (float)thumbnailWidth;
+									float heightFactor = bitmapImage.Height / (float)thumbnailWidth;
```

Comment:
I cannot see why this change is necessary

Reply:
For flipping itself it is not necessary, that's right.

But:
- The ffmpeg command specified the 16x16 size as string.
- I didn't like it when now in more places in other files a hardcoded assumption is made to the 16x16 size. Especially since e.g. also a feature request exists regarding parameterizable ffmpeg parameters.
- So I thought that the 16x16 size should be specified centrally in one place. But since both the 100:-1 scaling of the thumbnails and the 16:16 of the grayscale images run via one function, both were related to each other
- And on the other hand there is the tooltip zoom function of the thumbnails with a bad quality based on the low thumbnail resolution.
=> So I come to the idea, it would be the best solution, if the thumbnail and graybyte size would be parametrizable.
On the one hand it solves my original problem, having a clear central point of defining the size while on the other hand it would be the base of a future user configurable thumbnail size or a better zoom (not scaling tooltip image up but scaling thumbnail down based on tooltip image).
That is why I consistently exchanged  the thumbnail / graybyte size constants with the parameters.

Reply:
I don't comment the other places which also refer to this because it would be the same answer.
-> Central place of definition + flexibility to easily support other resolutions in future.
Maybe also for other graybyte resolutions, but this was not on focus. (So no focus on optimisation other resolutions)